e3/testsuite/__init__.py

```diff
@@ -445,26 +445,29 @@ def get_test_list(self, sublist):
         directory. If a test.yaml has a variants field, the test is expanded
         in several test, each test being associated with a given variant.
 
-        This function may be overriden. At this stage the self.global_env
+        This function may be overridden. At this stage the self.global_env
         (after update by the set_up method) is available.
 
         :param sublist: a list of tests scenarios or patterns
         :type sublist: list[str]
         :return: the list of selected test
         :rtype: list[str]
         """
-        # First retrive the list of test.yaml files
+        # First retrieve the list of test.yaml files
         result = [os.path.relpath(p, self.test_dir).replace('\\', '/')
                   for p in find(self.test_dir, 'test.yaml')]
         if sublist:
             logging.info('filter: %s' % sublist)
             filtered_result = []
             path_selectors = []
             for s in sublist:
-                subdir = os.path.relpath(os.path.abspath(s), self.test_dir)
-                if s.endswith('/') or s.endswith('\\'):
-                    subdir += '/'
-                path_selectors.append(subdir)
+                if os.path.exists(s):
```

Comment:
Can you add comments? Right now I don’t undertand even after reading the commit log and the patch, what the bug that is fixed is. Thanks!